reference/7.2/Microsoft.PowerShell.Core/About/about_FileSystem_Provider.md

```diff
@@ -86,11 +95,8 @@ name (`C:`, `D:`, ...) in the path.
 > [!NOTE]
 > PowerShell uses aliases to allow you a familiar way to work with provider
 > paths. Commands such as `dir` and `ls` are now aliases for
-> [Get-ChildItem](xref:Microsoft.PowerShell.Management.Get-ChildItem), `cd` is
-> an alias for
-> [Set-Location](xref:Microsoft.PowerShell.Management.Set-Location). and `pwd`
-> is an alias for
-> [Get-Location](xref:Microsoft.PowerShell.Management.Get-Location).
+> `Get-ChildItem`, `cd` is an alias for `Set-Location`. and `pwd` is an
+> alias for `Get-Location`.
```

Comment:
While we're in here, it might be worth clarifying that `ls` and `cd` are only aliases on Windows - on Linux/macOS they still use the appropriate native command.